src/app/(BottomTabNavigation)/Profile/DeleteAccount/styles.ts

```diff
@@ -6,13 +6,64 @@ export default StyleSheet.create({
   container: {
     flex: 1,
     backgroundColor: colors.white,
-    alignItems: 'center',
+    paddingHorizontal: 25,
```

Comment:
I'm assuming the padding is for content width? I think it'd be better to have a nested content container within this main container with a percentage width. You could also include the bottom buttons in this container.